storage/container_storage_interface/persistent-storage-csi-manila.adoc

```diff
@@ -12,17 +12,18 @@ toc::[]
 Familiarity with link:https://kubernetes.io/docs/concepts/storage/persistent-volumes/[PVs], link:https://kubernetes.io/docs/concepts/storage/persistent-volumes/#persistentvolumeclaims[persistent volume claims (PVCs)], link:http://blog.kubernetes.io/2016/10/dynamic-provisioning-and-storage-in-kubernetes.html[dynamic
 provisioning], and link:https://kubernetes.io/docs/admin/authorization/rbac/[RBAC authorization] is recommended.
 
-Before PVCs can be created, you must install the Manila CSI Driver Operator. The Operator creates the required storage classes for all available Manila share types needed for dynamic provisioning.
+To create CSI-provisioned PVs that mount to Manila storage assets, {product-title} installs the Manila CSI Driver Operator and the Manila CSI driver by default in the `openshift-cluster-csi-drivers` namespace.
 
-After the Operator is installed, you must also create the ManilaDriver Custom Resource (CR) that is required in the {product-title} cluster.
+* The **Manila CSI Driver Operator** creates the required storage classes that are needed to create PVCs for all available Manila share types.
 
-include::modules/persistent-storage-csi-manila-install-operator.adoc[leveloffset=+1]
+* The **Manila CSI driver** is a custom resource (CR) that enables you to create and mount Manila PVs.
 
-include::modules/persistent-storage-csi-manila-install-driver.adoc[leveloffset=+1]
+== About CSI
+Storage vendors have traditionally provided storage drivers as part of Kubernetes. With the implementation of the Container Storage Interface (CSI), third-party providers can instead deliver storage plug-ins using a standard interface without ever having to change the core Kubernetes code.
 
-include::modules/persistent-storage-csi-manila-dynamic-provisioning.adoc[leveloffset=+1]
+CSI Operators give {product-title} users storage options, such as volume snapshots, that are not possible with in-tree volume plug-ins.
 
-include::modules/persistent-storage-csi-manila-uninstall-operator.adoc[leveloffset=+1]
+include::modules/persistent-storage-csi-manila-dynamic-provisioning.adoc[leveloffset=+1]
```

Comment:
@tombarron @Fedosin - Could you PTAL at where I've made updates for 4.6 (OLM -> CSO Manila Operator) and let me know if I've missed anything from your perspective?